nova/virt/vmwareapi/cluster_util.py

```diff
@@ -167,15 +168,33 @@ def update_vm_group_membership(session, cluster, vm_group_name, vm_ref):
     operation = None
     group = _get_vm_group(cluster_config, vm_group_name)
 
+    operation = None
```

Comment:
`operation` is already `None` 3 lines up.

Reply:
I removed both and moved each operation to the end of their respective blocks. That also should make clear that it is defined in every branch.